sos/plugins/manageiq.py

```diff
@@ -84,4 +84,9 @@ def setup(self):
                     os.path.join(pg_dir, 'postgresql.conf')
             ])
 
+        self.collectExtOutput([
+            "rpm -V cfme cfme-appliance cfme-gemset
```

Comment:
This is what the `--verify` option is for. Add the package names to the `packages` class member and when `--verify` is used, the package manager defined by the policy will run a verify command against all packages listed.

Reply:
Thanks for the feedback, made the change as requested.